### ISSUE TITLE
Fix Refresh on Pull with ListView

### ISSUE THREAD

**Issue by PureWeen** [dotnet/maui#4196]
### Description

Pull to Refresh looks to not be working for iOS/WinUI (haven't tested Android)

### Steps to Reproduce

Run the ListViewRefresh sample

### Version with bug

Preview 13

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Windows

### Affected platform versions

Android, iOS 15

**Comment by XamlTest** [comment]
Thank you so much for your bug report! Unfortunately, we cannot reproduce your issue with the information above, could you provide a sample project and screenshots or more details that we can reproduce your issue? That should greatly speed up the process, thanks!

**Comment by PureWeen** [comment]
@XamlTest the repro for this issue is part of the Sample gallery that comes with the Maui source code. Don't worry about validating/marking this issue as needs-repro. Thank you!!

**Comment by shimingsg** [comment]
still repro on Android with this https://github.com/dotnet/maui/blob/main/src/Controls/samples/Controls.Sample/Pages/Compatibility/ListViewGalleries/ListViewRefresh.cs 
 VS: 17.3.0 Preview 2.0 [32506.315.main]
can't pull on Windows with mouse since I don't have touch device.

**Comment by shimingsg** [comment]
at the same time ,I found the buttons in headerGrid are disappeared after scrolling with mouse scroll wheel
![4196](https://user-images.githubusercontent.com/20982341/167366804-2ac79646-de3d-45fa-8b70-b8dda2b5298b.gif)
@PureWeen, is this a new issue? Android is Emulator.

**Comment by veringm** [comment]
Any updates on this one?

**Comment by bronteq** [comment]
ListView will be deprecated in Net10 https://github.com/dotnet/maui/issues/28699

**Comment by jfversluis** [comment]
Indeed, ListView is deprecated, I don't think we'll be working on this anymore. Thank you!
